source/usage-examples/find.txt

```diff
@@ -3,3 +3,54 @@ Find Multiple Documents
 =======================
 
 .. default-domain:: mongodb
+
+You can find multiple documents in a collection by using the ``Find()``
+method.
+
+The following example passes a query filter to the ``Find()`` method,
+which matches documents in the ``zips`` collection if the value of their
+``pop`` field is less than or equal to *500*:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/find.go
+   :start-after: begin find
+   :end-before: end find
+   :emphasize-lines: 4
+   :language: go
+   :dedent:
+
+Click here **<TODO: github link to file>** to see a fully runnable example.
+
+Expected Result
+---------------
+
+This example returns a ``Cursor`` object that contains
+documents similar to these:
+
+.. code-block:: json
+   :copyable: false
+
+   // results truncated
+   ...
+   { ... , "city" : "SHAKTOOLIK", "zip" : "99771", ... , "pop" : 183, "state" : "AK" },
+   { ... , "city" : "DARLINGTON", "zip" : "83231", ... , "pop" : 12, "state" : "ID" },
+   { ... , "city" : "EAST BARRE", "zip" : "05649", ... , "pop" : 381, "state" : "VT" },
```

Comment:
```suggestion
   { ...  "pop" : 183, "state" : "AK" },
   { ...  "pop" : 12, "state" : "ID" },
   { ...  "pop" : 381, "state" : "VT" },
```

Suggestion: Could the return value be shortened? I think the important thing is the population.

Nit: I think having two sets of ellipses in each sample return value is hard to read.

Reply:
To allow readers to locate these documents to make sure the code worked, I think the city, state pair is necessary to disambiguate. Since the city names are also the same number of chars, all the fields are pretty much aligned anyways, making the results easy to visually process in my opinion. Omitted the zip field to eliminate horizontal scroll.

Reply:
Why is city state pair the best pair for disambiguating these items? Does City and Population (or state and population) have a collision?